platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -341,7 +341,7 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
 
     private class FeaturesPathHandler(
         private val context: Context,
-        private val featureFile: ZipFile
+        private val featureFilePath: String
```

Comment:
Ah, OK, I see. featureFile no longer uses a ZipFile. However, it's not a path either. `PathName`?

Reply:
It depends, a string can also represent a path, it does not necessarily have to be of type "Path".